src/desktop/apps/order2/index.tsx

```diff
@@ -1,32 +1,9 @@
-import React from 'react'
+import * as routes from 'desktop/apps/order2/routes'
+import adminOnly from 'desktop/lib/admin_only.coffee'
```

Comment:
also, this keeps giving me a typescript error that it can't find the module (although it _can_, and it does work)

Reply:
@damassi do you have any ideas on ☝️?

Reply:
Oh sorry! i was looking on my phone and saw the `fix ts errors` commit and assumed this was addressed. Let pull down and take a look. I think it could be a couple things but need to verify.

Reply:
This has to do with `.coffee` extensions and TypeScript's inability to understand them natively. Since everything is piped through node `register` hooks compilation happens as it should, but in terms of type-checking this wont work. Might be good to quickly convert `admin_only` into a JS file and deprecate the coffee one? That would fix the issue.